tests/integration/models/model_test.py

```diff
@@ -20,7 +20,15 @@
 import os
 
 from django.db import connection
-from django.db.models import get_models
+
+try:
+    # Django >= 1.8
```

Comment:
Same reason as the previous case: 1.8 is the first version where the new method works, and the last method where the old method works. In 1.7 I think there is a warning to this effect.

IIRC we used to use the old method also on 1.8 but I switched the order so that we always prefer the new method whenever possible.